src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/ReflectedMethodNode.cs

```diff
@@ -64,14 +64,17 @@ public override IEnumerable<DependencyListEntry> GetStaticDependencies(NodeFacto
             {
                 if (_method.IsVirtual)
                 {
+                    // Virtual method use is tracked on the slot defining method only.
+                    MethodDesc slotDefiningMethod = MetadataVirtualMethodAlgorithm.FindSlotDefiningMethodForVirtualMethod(_method);
                     if (_method.HasInstantiation)
                     {
-                        dependencies.Add(factory.GVMDependencies(_method.GetCanonMethodTarget(CanonicalFormKind.Specific)), "GVM callable reflectable method");
+                        // FindSlotDefiningMethod might uninstantiate. We might want to fix the method not to do that.
+                        if (slotDefiningMethod.IsMethodDefinition)
```

Comment:
I definitely don't understand what "uninstantiate" means here.

Reply:
Might remove the instantiation of the generic method. Sometimes. I think we just don't call this method with generic methods. I didn't want to fix this here because we call this method in two dozen places and it would require carefully reviewing each. So we just instantiate the method back if it became uninstantiated.